src/Symfony/Component/Console/Command/Command.php

```diff
@@ -656,6 +669,8 @@ public function getHelper($name)
      *
      * It must be non-empty and parts can optionally be separated by ":".
      *
+     * @return void
+     *
      * @throws InvalidArgumentException When the name is invalid
      */
     private function validateName(string $name)
```

Comment:
```suggestion
     * @throws InvalidArgumentException When the name is invalid
     */
    private function validateName(string $name): void
```

For private methods, you can safely add return type declarations.

Reply:
Oops, didn't notice the `private`, thanks!